### ISSUE TITLE
[BUG] Install of analysis-icu plugin fails in beta1

### ISSUE THREAD

**Issue by janhoy** [opensearch-project/opensearch-build#180]
**Describe the bug**
Trying to do install analysis-icu plugin from docker container fails with HTTP 403

**To Reproduce**
1. `docker run -ti opensearchproject/opensearch:1.0.0-beta1 bash`
2. `bin/opensearch-plugin install analysis-icu`
3. See error

```
-> Installing analysis-icu
-> Downloading analysis-icu from opensearch
-> Failed installing analysis-icu
-> Rolling back analysis-icu
-> Rolled back analysis-icu
Exception in thread "main" java.io.IOException: Server returned HTTP response code: 403 for URL: https://artifacts.opensearch.org/releases/plugins/analysis-icu/1.0.0/analysis-icu-1.0.0-beta1.zip
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1932)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1528)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
	at org.opensearch.plugins.InstallPluginCommand.downloadZip(InstallPluginCommand.java:438)
	at org.opensearch.plugins.InstallPluginCommand.downloadAndValidate(InstallPluginCommand.java:515)
	at org.opensearch.plugins.InstallPluginCommand.download(InstallPluginCommand.java:306)
	at org.opensearch.plugins.InstallPluginCommand.execute(InstallPluginCommand.java:260)
	at org.opensearch.plugins.InstallPluginCommand.execute(InstallPluginCommand.java:237)
	at org.opensearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:99)
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:140)
	at org.opensearch.cli.MultiCommand.execute(MultiCommand.java:104)
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:140)
	at org.opensearch.cli.Command.main(Command.java:103)
	at org.opensearch.plugins.PluginCli.main(PluginCli.java:60
```

**Expected behavior**
Plugin installed

**Plugins**
(none)

**Screenshots**
(None)

**Host/Environment (please complete the following information):**
Docker (macos)

**Additional context**
(none)

**Comment by dblock** [comment]
I moved this into opensearch-build because it looks like an unpublished artifact.

**Comment by peternied** [comment]
This will be fixed in the v1.1.0 release https://github.com/opensearch-project/OpenSearch/issues/996

**Comment by bbarani** [comment]
We will fix this issue as part of this PR - https://github.com/opensearch-project/opensearch-build/issues/86
